diff -x config.log -x config.status -ru tmp/webstump/Makefile work/webstump/Makefile
--- tmp/webstump/Makefile	1999-05-06 10:40:31.000000000 -0700
+++ work/webstump/Makefile	2025-07-30 22:01:57.661659457 -0700
@@ -1,7 +1,7 @@
 
 # Edit these variables
 
-WEBSTUMP_HOME = /home/ichudov/public_html/stump/webstump
+WEBSTUMP_HOME = ../webstump
 CC = cc
 
 # do not edit below
diff -x config.log -x config.status -ru tmp/webstump/src/wrapper.c work/webstump/src/wrapper.c
--- tmp/webstump/src/wrapper.c	1999-05-06 10:40:32.000000000 -0700
+++ work/webstump/src/wrapper.c	2025-07-30 21:51:35.714854004 -0700
@@ -11,6 +11,8 @@
 #include <stdio.h>
 #include <sys/types.h>
 #include <sys/stat.h>
+#include <string.h>
+#include <unistd.h>
 
 const char * webstump_home = WEBSTUMP_HOME;
 
