diff -x config.log -x config.status -ru tmp/idutils-4.6/lib/fseeko.c work/idutils-4.6/lib/fseeko.c
--- tmp/idutils-4.6/lib/fseeko.c	2012-01-28 08:26:04.000000000 -0800
+++ work/idutils-4.6/lib/fseeko.c	2024-01-22 15:48:05.804816264 -0800
@@ -90,7 +90,7 @@
       && fp->__put_limit == fp->__bufp
       && !fp->__pushed_back)
 #else
-  #error "Please port gnulib fseeko.c to your platform! Look at the code in fpurge.c, then report this to bug-gnulib."
+  #warning "Please port gnulib fseeko.c to your platform! Look at the code in fpurge.c, then report this to bug-gnulib."
 #endif
     {
       /* We get here when an fflush() call immediately preceded this one (or
diff -x config.log -x config.status -ru tmp/idutils-4.6/lib/fseterr.c work/idutils-4.6/lib/fseterr.c
--- tmp/idutils-4.6/lib/fseterr.c	2012-01-28 08:22:33.000000000 -0800
+++ work/idutils-4.6/lib/fseterr.c	2024-01-22 15:53:50.810752644 -0800
@@ -71,6 +71,6 @@
     }
   errno = saved_errno;
 #else
- #error "Please port gnulib fseterr.c to your platform! Look at the definitions of ferror and clearerr on your system, then report this to bug-gnulib."
+ #warning "Please port gnulib fseterr.c to your platform! Look at the definitions of ferror and clearerr on your system, then report this to bug-gnulib."
 #endif
 }
diff -x config.log -x config.status -ru tmp/idutils-4.6/lib/stdio.in.h work/idutils-4.6/lib/stdio.in.h
--- tmp/idutils-4.6/lib/stdio.in.h	2012-01-28 08:26:05.000000000 -0800
+++ work/idutils-4.6/lib/stdio.in.h	2024-01-18 18:14:29.284982198 -0800
@@ -715,7 +715,6 @@
 /* It is very rare that the developer ever has full control of stdin,
    so any use of gets warrants an unconditional warning.  Assume it is
    always declared, since it is required by C89.  */
-_GL_WARN_ON_USE (gets, "gets is a security hole - use fgets instead");
 #endif
 
 
