diff -x config.log -x config.status -ru tmp/archimedes-2.0.1/src/constants.h work/archimedes-2.0.1/src/constants.h
--- tmp/archimedes-2.0.1/src/constants.h	2011-08-26 06:43:23.000000000 -0700
+++ work/archimedes-2.0.1/src/constants.h	2022-05-20 19:10:15.155543727 -0700
@@ -25,7 +25,8 @@
 
 // ######################################################
 // Created on 5 aug.2004, Siracusa, J.M.Sellier
-// Last modif. : 17 aug.2011, Carry le Rouet, France, J.M.Sellier
+// modif. : 17 aug.2011, Carry le Rouet, France, J.M.Sellier
+// May 2022 : update using Codata https://physics.nist.gov/cuu/Constants/Table/allascii.txt, CH
 // ######################################################
 
 // Universal Physical Constants in M.K.S.C. system
@@ -34,32 +35,35 @@
 // ######################################################
 
 // Boltzmann constant (Joule/Kelvin)
- const real KB=1.380658e-23;
+ const real KB=1.380649e-23;
 // Electron charge in absolute value (Coulomb)
- const real Q=1.60217733e-19;
+ const real Q=1.602176634e-19;
 // Reduced Planck constant (Joule*sec)
- const real HBAR=1.05457266e-34;
+ const real HBAR=1.054571817646e-34;
 // Permittivity of free space (F/m)
- const real EPS0=8.854187817e-12;
+ const real EPS0=8.8541878128e-12;
 // Electron Mass (Kg)
- const real M=9.1093897e-31;
+ const real M=9.1093837015e-31;
+
 // Silicon intrinsic density for room temperature
  const real NI=1.45e16;
+
 // Pi number
- const real PI=3.141592654;
+ const real PI=3.14159265358979323846264338;
+
 // electron energy step (eV) for the MC method
  const real DE=0.002;
 // Silicon low field mobility (m^2/(V*sec))
- real MIU0=1400.e-4;
+ const real MIU0=1400.e-4;
 // Silicon saturation velocity (m/sec)
- real VS=1.e5;
+ const real VS=1.e5;
 // Silicon heavy hole effective mass
- real mstarhole=0.57;
+ const real mstarhole=0.57;
 // Silicon heavy hole low field mobility (m^2/(V*sec))
- real MIU0hole=0.0471;
+ const real MIU0hole=0.0471;
 // Silicon heavy hole saturation velocity (m/sec)
- real VShole=1.e5;
+ const real VShole=1.e5;
 // Silicon Schottky contact density (1/m^3)
- real NGATE=3.9e11;
+ const real NGATE=3.9e11;
 // ######################################################
 // ######################################################
