diff -x config.log -x config.status -ru tmp/ZRTPCPP-4.7.0/cryptcommon/ZrtpRandom.h work/ZRTPCPP-4.7.0/cryptcommon/ZrtpRandom.h
--- tmp/ZRTPCPP-4.7.0/cryptcommon/ZrtpRandom.h	2020-02-01 10:04:00.000000000 -0800
+++ work/ZRTPCPP-4.7.0/cryptcommon/ZrtpRandom.h	2024-03-07 22:16:58.605786768 -0800
@@ -24,6 +24,7 @@
  * @{
  */
 
+#include <stdbool.h>
 #include <string.h>
 #if !(defined(_WIN32) || defined(_WIN64))
 #include <unistd.h>
@@ -82,4 +83,4 @@
 }
 #endif
 
-#endif /* ZRTPRANDOM */
\ No newline at end of file
+#endif /* ZRTPRANDOM */
